remote.lua

```diff
@@ -33,12 +33,18 @@ end
 
 --
 -- Creates a sensor managed by a remote interface (another mod or script)
+-- mod_name: Name of the mod registering the sensor. Sensor will be removed if the mod is removed from the game.
 -- sensor_name: internal name of the sensor. Should be unique.
 -- sensor_text: Text to display in the active gui
 -- sensor_caption: Sensor setting name in the EvoGUI settings panel
 -- sensor_color: Font color of the text to display in the active gui, optional, may be nil
 -- example: remote.call("EvoGUI", "create_remote_sensor", "mymod_my_sensor_name", "Text: Lorem Ipsum", "[My Mod] Lorem Ipsum Text")
-local function create_remote_sensor(sensor_name, sensor_text, sensor_caption, sensor_color)
+local function create_remote_sensor(mod_name, sensor_name, sensor_text, sensor_caption, sensor_color)
+    if not mod_name then
+        evogui.log({"err_nomodname"})
```

Comment:
Given that parameters are not named, and but instead are ordered, if mod_name is nil, then so are all the other parameters.

I did consider turning this into a table parameter that had named fields. The advantage would be that you could add more parameters w/o order mattering or breaking the API in the future, but the disadvantage is extra complexity. I didn't think it was worth the complexity...yet.